src/main/java/studos/logic/LoginLogic.java

```diff
@@ -146,14 +145,19 @@ public void handle(final MouseEvent event) {
         textRights.textProperty()
             .bind(I18N.createStringBinding("login.Rights"));
 
+        if (I18N.getDefaultLocale().getLanguage().equals("pl")) {
+            polishLanguageButton.setSelected(true);
+        } else {
+            englishLanguageButton.setSelected(true);
+        }
```

Comment:
This will works perfectly for 2 languages. 

Question for future: There is no other way to select the button than "else" ?